prusti-viper/src/encoder/snapshot/encoder.rs

```diff
@@ -912,8 +912,8 @@ impl SnapshotEncoder {
 
                 Ok(Snapshot::Array {
                     predicate_type: predicate_type.convert_to_snapshot(),
-                    domain: self.insert_domain(domain),
-                    snap_func: self.insert_function(snap_func),
+                    _domain: self.insert_domain(domain),
```

Comment:
If these are not used at all, we can remove the fields.

Reply:
I will refactor that code later once I implement `vir-high`.